docs/documentation/server_admin/topics/user-federation/sssd.adoc

```diff
@@ -114,83 +114,64 @@ This script makes the following changes to `/etc/sssd/sssd.conf`:
+dbus-send --print-reply --system --dest=org.freedesktop.sssd.infopipe /org/freedesktop/sssd/infopipe org.freedesktop.sssd.infopipe.GetUserAttr string:<username> array:string:mail,givenname,sn,telephoneNumber
+
+dbus-send --print-reply --system --dest=org.freedesktop.sssd.infopipe /org/freedesktop/sssd/infopipe org.freedesktop.sssd.infopipe.GetUserGroups string:<username>
 ----
 +
-If the setup is successful, you see the user's group. If this command returns a timeout or an error, the federation provider running on {project_name} cannot retrieve any data. This error usually happens because the server is not enrolled in the FreeIPA IdM server, or does not have permission to access the SSSD service.
+If the setup is successful, each command displays the user's attributes and groups respectively. If there is a timeout or an error, the federation provider running on {project_name} cannot retrieve any data. This error usually happens because the server is not enrolled in the FreeIPA IdM server, or does not have permission to access the SSSD service.
 +
 If you do not have permission to access the SSSD service, ensure that the user running the {project_name} server is in the `/etc/sssd/sssd.conf` file in the following section:
 +
 [source,bash,subs=+attributes]
 ----
 [ifp]
-allowed_uids = root, your_username
+allowed_uids = root, yourOSUsername
 ----
-
-==== Enabling the SSSD federation provider
-
-{project_name} uses DBus-Java to communicate at a low level with D-Bus. D-Bus depends on the http://www.matthew.ath.cx/projects/java/[Unix Sockets Library].
-
-ifeval::[{project_community}==true]
-
-You can find an RPM for this library in https://github.com/keycloak/libunix-dbus-java/releases[the keycloak repository]. Before installing this RPM, check the RPM signature using this command:
-
++
+And the `ipaapi` system user is created inside the host. This user is necessary for the `ifp` service. Check the user is created in the system.
++
 [source,bash,subs=+attributes]
 ----
-  $ rpm -K libunix-dbus-java-0.8.0-1.fc24.x86_64.rpm
-  libunix-dbus-java-0.8.0-1.fc24.x86_64.rpm:
-    Header V4 RSA/SHA256 Signature, key ID 84dc9914: OK
-    Header SHA1 digest: OK (d17bb7ebaa7a5304c1856ee4357c8ba4ec9c0b89)
-    V4 RSA/SHA256 Signature, key ID 84dc9914: OK
-    MD5 digest: OK (770c2e68d052cb4a4473e1e9fd8818cf)
+grep ipaapi /etc/passwd
+ipaapi:x:992:988:IPA Framework User:/:/sbin/nologin
 ----
 
-Install the RPM using this command:
+==== Enabling the SSSD federation provider
 
-[source,bash,subs=+attributes]
-----
-$ sudo yum install libunix-dbus-java-0.8.0-1.fc24.x86_64.rpm
-----
+{project_name} uses https://github.com/hypfvieh/dbus-java[DBus-Java] project to communicate at a low level with D-Bus and https://github.com/java-native-access/jna[JNA] to authenticate via Operating System Pluggable Authentication Modules (PAM).
 
+ifeval::[{project_community}==true]
```

Comment:
One minor note: Do we need this `ifeval` or can we remove it and keep the note about Java 17 always in the docs?

It seems the product would be supported just for Java 17, however in theory, you never know if something changes (or there are any support exceptions etc)? WDYT?

Reply:
Yep, it's OK for me. I just differentiate between community and product to not mention jdk-17 in product if jdk-17 was compulsory. It's done now.